### PR TITLE
Fixed Mustache

### PR DESCRIPTION
**Pull request by tolijoonbug**

### CODE REVIEW

File: lib/Sky/Api/Resource.php [hunk 1]
```diff
@@ -52,6 +52,12 @@
     protected static $possible_errors = array();
 
     /**
+     * Errors this has
+     * @var array 
+     */
+    public $errors = array();
```
Comment (stanistan) [inline lib/Sky/Api/Resource.php]: this should be protected and not public

Comment (tolijoonbug) [inline lib/Sky/Api/Resource.php]: done

File: lib/Sky/Mustache.php [hunk 2]
```diff
@@ -129,20 +129,36 @@ private function getPartials($markup, $paths = array(), $partials = array())
...
             }
         }
         return $partials;
     }
+    
+    /**
+     * Checks if markup is a mustache
+     * @param string $markup 
+     */
+    private function isMustache($markup){
```
Comment (stanistan) [inline lib/Sky/Mustache.php]: @return boolean

opening brace, use consistent spacing in the if / else if { } blocks

Comment (tolijoonbug) [inline lib/Sky/Mustache.php]: done

**Pull request merged**